package.json

```diff
@@ -80,33 +80,33 @@
     "react-dom": "17.0.2",
     "react-dropzone": "^11.4.2",
     "react-helmet": "6.1.0",
-    "react-hook-form": "^7.20.2",
-    "react-i18next": "^11.14.2",
+    "react-hook-form": "^7.20.4",
+    "react-i18next": "^11.14.3",
     "react-leaflet": "^3.2.2",
     "react-share": "^4.4.0",
-    "react-spring": "^9.3.1",
+    "react-spring": "^9.3.2",
     "react-use": "^17.3.1",
     "rehype-react": "^7.0.3",
-    "sass": "^1.43.4",
-    "slugify": "^1.6.2",
+    "sass": "^1.43.5",
+    "slugify": "^1.6.3",
     "styled-components": "^5.3.3",
     "unified": "^10.1.1",
-    "webpack": "^5.64.2",
+    "webpack": "^5.64.4",
     "what-input": "^5.2.10"
   },
   "devDependencies": {
     "@babel/core": "~7.16.0",
-    "@storybook/addon-actions": "^6.3.12",
-    "@storybook/addon-essentials": "^6.3.12",
-    "@storybook/addon-links": "^6.3.12",
-    "@storybook/builder-webpack5": "^6.3.12",
-    "@storybook/manager-webpack5": "^6.3.12",
-    "@storybook/react": "^6.3.12",
-    "@testing-library/jest-dom": "^5.15.0",
+    "@storybook/addon-actions": "^6.4.0",
+    "@storybook/addon-essentials": "^6.4.0",
+    "@storybook/addon-links": "^6.4.0",
+    "@storybook/builder-webpack5": "^6.4.0",
+    "@storybook/manager-webpack5": "^6.4.0",
+    "@storybook/react": "^6.4.0",
+    "@testing-library/jest-dom": "^5.15.1",
```

Comment:
Interesting https://storybook.js.org/releases/6.4 mentions interactive testing. There is no link and I found this:
https://storybook.js.org/blog/interactive-stories-beta/.

You can define what elements should be clicked in order to automate the manual steps of checking if everything works in case of more complex components. Imagine a form we can fill out automatically like you know it from e2e tests but in storybook